package.json

```diff
@@ -114,6 +114,7 @@
     "lodash": "4.17.21",
     "ol": "8.2.0",
     "punycode": "^2.1.1",
+    "rc-slider": "^10.5.0",
```

Comment:
I think we should make it a policy to kill the chevron syntax for when we add and update packages moving forwards. I just do not trust developers to abide by npm semvar's definition for it ever to be worth the risk for the tiny tiny tiny minute advantage it supposedly adds.